assets/scripts/segments/__tests__/Segment.test.js

```diff
@@ -203,7 +205,7 @@ describe('Segment', () => {
         }
       )
       setLastStreet() // ToDo: needs to be refactored
-      fireEvent.mouseOver(screen.getByTestId('segment'))
+      userEvent.hover(screen.getByTestId('segment'))
```

Comment:
There does seem to be a problem with converting to `userEvent` for the delete keys, here and in the test below. 

I've traced the issue to the library's implementation of the delete key, which assumes that you're pressing it on a text string. I am [opening this issue](https://github.com/testing-library/user-event/issues/506) to ask them if there is a different way to achieve this or if the library itself should be updated to handle this use case.

Reply:
Looks like it would be some desired functionality to add. Should we leave it here for now? I wouldn't mind helping them with the contrib if we had the time, but that seems like it's own rabbithole

Reply:
Yes, let's leave `fireEvent` for now. Let's also add a comment here that references this issue.